executor.go

```diff
@@ -427,6 +461,22 @@ func (e *Executor) runPass() {
 	e.mu.Lock()
 	defer e.mu.Unlock()
 
+	// If "--help/-h" is passed, show help text for first task
```

Comment:
do we want to support showing help text for multiple tasks if they all invoke the help flag? maybe that behavior is more intuitive?

Reply:
New behaviour:
`silly/mock/task --help fun/mock/task --help goofy/mock/task --help`
![Screenshot 2023-01-30 at 6 21 34 PM](https://user-images.githubusercontent.com/16676111/215626627-750abc00-d893-428b-abb9-24ca01b97aca.png)